server/ott-config.ts

```diff
@@ -487,6 +488,26 @@ export function loadConfigFile() {
 	conf.validate({ allowed: "warn" });
 }
 
+export function validateConfig() {
+	const mailEnabled = conf.get("mail.enabled");
+	const mailResult = validateMail();
+	const confResult = intoResult(() => conf.validate({ allowed: "strict" }));
+
+	if ((!mailResult.ok && mailEnabled) || !confResult.ok) {
+		if (!mailResult.ok) {
+			log.error(mailResult.value.message);
+		}
+
+		if (!confResult.ok) {
+			log.error(confResult.value.message);
+		}
+
+		process.exit(1);
```

Comment:
On it's own, `validateConfig()` does not make it clear to the reader that calling it will make the process exit. Move the calls to `process.exit()` to `app.ts`